drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -5055,6 +5055,7 @@ RID RenderingDeviceVulkan::shader_create_from_bytecode(const Vector<uint8_t> &p_
 
 	Shader *shader = shader_owner.get_or_null(id);
 
+	ERR_FAIL_NULL_V(shader, RID());
```

Comment:
```suggestion
	Shader *shader = shader_owner.get_or_null(id);
	ERR_FAIL_NULL_V(shader, RID());

```

Validation usually comes right after getting, not just before using.